app/models/publishable.rb

```diff
@@ -72,10 +98,12 @@ def publish_to_portals(self_url)
   def self.included(clazz)
     clazz.class_eval do
       after_update :auto_publish_to_portal
+      has_many :portal_publications, :as => :publishable, :order => :updated_at
     end
   end
 
   def auto_publish_to_portal
+    hash =
```

Comment:
Sorry, that 'hash =' was introduced when I was fixing the merge error I saw after the tests ran on Travis and I was switching branches back and forth and using undo/redo in my editor to apply changes between branches.

Sent from my iPhone

> On Jul 7, 2015, at 5:15 PM, Scott Cytacki <notifications@github.com> wrote:
> 
> In app/models/publishable.rb:
> 
> >      end
> >    end
> >  
> >    def auto_publish_to_portal
> > +    hash =
> this looks like an error, ruby probably doesn't complain because it sets the value of hash to the return value of logger
> 
> —
> Reply to this email directly or view it on GitHub.
>